yarn.lock

```diff
@@ -87,7 +87,7 @@
   resolved "https://registry.yarnpkg.com/@artsy/palette-tokens/-/palette-tokens-4.0.0.tgz#ee0d157ad73ca0e2e252f896a21fba798b20d4d9"
   integrity sha512-Os2fimLkHgxWLZZ8xRPWm4t3mBTMMBsKO+Iqgn2d44VjhcN7w9XxAp8VK8sRSLJmcrajnTczZZtFy273lYeqaA==
 
-"@artsy/palette@24.5.1":
+"@artsy/palette@24.5.1", "@artsy/palette@^24.5.1":
```

Comment:
@dzucconi Still seeing this dupe here. Any ideas why?

Reply:
Actually that was it being deduped! https://github.com/artsy/force/pull/11201 wasn't merged. Anyway; all good now.